packages/angular_devkit/schematics/tasks/index.ts

```diff
@@ -5,8 +5,10 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-export { NodePackageInstallTask } from './node-package/install-task';
-export { NodePackageLinkTask } from './node-package/link-task';
+// TODO: This folder should not be prefixed with `not-`, but its's needed to workaround a Bazel bug:
+// https://github.com/bazelbuild/rules_nodejs/issues/313
+export { NodePackageInstallTask } from './not-node-package/install-task';
+export { NodePackageLinkTask } from './not-node-package/link-task';
```

Comment:
This is quite invasive.  Is there any chance we can prioritize a fix for this upstream?

Reply:
Looping in @gregmagolan to see what he thinks.

Reply:
Depending on the timeframe for an upstream fix, can we change `not-node-package` to `package-manager` instead?  This could potentially be a permanent change since the name still reflects the purpose of the directory contents.

Reply:
SGTM, changed it.